docs/using-airbyte/core-concepts/sync-modes/full-refresh-overwrite-deduped.md

```diff
@@ -0,0 +1,50 @@
+---
+products: all
+---
+
+# Full Refresh - Overwrite + Deduped
+
+## Overview
+
+The **Full Refresh** modes are the simplest methods that Airbyte uses to sync data, as they always retrieve all available information requested from the source, regardless of whether it has been synced before. This contrasts with [**Incremental sync**](./incremental-append.md), which does not sync data that has already been synced before.
+
+In the **Overwrite + Deduped** variant, new syncs will replace all data in the existing destination table and then pull the new data in. Therefore, data that has been removed from the source after an old sync will be deleted in the destination table. The **deduped** variant also means that the data in the final table will be unique per primary key \(unlike [Full Refresh Overwrite](./full-refresh-overwrite.md)\). This is determined by sorting the data using the cursor field and keeping only the latest de-duplicated data row.
+
```

Comment:
Can we add a quick sentence here about "As Full Refresh syncs are Resumable as much as possible, we recommend Full Refresh Overwrite Dedup if avoiding duplicates is a strong requirement. Bear in mind this incurs additional warehouse costs.', and linking the Resumable doc page in there.

Something like this!